src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day calendar--month-length_31">
```

Comment:
```suggestion
      class="calendar calendar--start-day_wed calendar--month-length_31">
```
set some default start day